js/src/scatter.ts

```diff
@@ -335,6 +432,10 @@ class ScatterView extends widgets.WidgetView {
         const current  = new values.Values(scalar_names, [], this.get_current.bind(this), sequence_index, vector4_names);
         const previous = new values.Values(scalar_names, [], this.get_previous.bind(this), sequence_index_previous, vector4_names);
 
+        // Workaround for shader issue - color redefine 
+        current.ensure_array(["color"]);
```

Comment:
I think that anwsers a previous comment, maybe also put it in the shader code, sth like `// threejs already uses the name color `